diff --git demo/image_classification/image_provider.py demo/image_classification/image_provider.py
index 6a315ff..2d6afc3 100644
--- demo/image_classification/image_provider.py
+++ demo/image_classification/image_provider.py
@@ -74,7 +74,8 @@ def processData(settings, file_list):
                 indexes = list(range(len(data['images'])))
                 if settings.is_train:
                     random.shuffle(indexes)
-                for i in indexes:
+                for cnt, i in enumerator(indexes):
+                    if cnt == 1000: break
                     if settings.use_jpeg == 1:
                         img = image_util.decode_jpeg(data['images'][i])
                     else:
@@ -87,3 +88,4 @@ def processData(settings, file_list):
                         'image': img_feat.astype('float32'),
                         'label': int(label)
                     }
+
